### PR TITLE
example: Add Sierpinski sieve

### LINKED ISSUE #29: Sierpinski triangle example
It's a common example ever since it was used as a demo for react fiber. We should have an implementation.

### PR DESCRIPTION
**Pull request by sseg**
Adds a Sierpinski triangle example based on the React Fiber demo: https://github.com/claudiopro/react-fiber-vs-stack-demo

Resolves #29 

Feedback would be greatly appreciated. I think I've got a sense of how the dodrio API is intended to be used but if there's anything that could change to make this example more idiomatic I'd be glad to adjust it.

### CODE REVIEW

**Comment by sseg** [comment]
An important aspect of the React example is the injection of work on the main thread (e.g. https://github.com/claudiopro/react-fiber-vs-stack-demo/blob/58850495f448ecbf9b109b86a7900a844fd06235/fiber.html#L94-L99)

I didn't go ahead and emulate this in the example since long-running work is intended to be more easily relegated to web workers with Rust/WASM.

File: examples/sierpinski-triangle/index.html [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+  <script>
+    wasm_bindgen("pkg/dodrio_sierpinski_bg.wasm");
+  </script>
+
+  <script>
+    let lastTime = 0;
+    let frameCount = 0;
+    const fps = document.getElementById('fps');
+
+    function updateFPS(time) {
```
Comment (MichalObi) [inline examples/sierpinski-triangle/index.html:52]: How about more es6 style like: 
`const updateFPS = time => {...}`

**Comment by fitzgen** [comment]
Thanks! For the future, allocating a bunch of vecs and then immediately getting rid of them isn't the fastest thing in the world, and it is often nicer to just pass a mutable vec as a parameter: https://github.com/fitzgen/dodrio/commit/87d4f247741f791081b9c36ee713048016d126ce

**Comment by sseg** [comment]
Noted! Thanks for the fix-up

**Pull request merged**